src/main/java/com/xiao/users/controller/UserController.java

```diff
@@ -54,4 +54,11 @@ public ResponseEntity<Page<UserDto>> findAllUser(
                 .body(userDtos);
     }
 
+    @PutMapping(value = "/{id}")
+    public ResponseEntity<UserDto> updateUser(@PathVariable Long id,@Valid @RequestBody UserDto userDto){
```

Comment:
In the UserDto, we have validation for all fields. 
However, in each update case, we don't update all fields of the user, correct?